app/controllers/diagnostics_controller.rb

```diff
@@ -8,7 +8,7 @@ def set_content_type
   def move
     move = Move.accessible_by(current_ability).find_by(id: params[:id]) || Move.accessible_by(current_ability).find_by(reference: params[:id])
     # NB: personal details should only be available on localhost, dev, staging and uat; not on pre-prod or production
-    include_person_details = Rails.env.development? || ENV.fetch('HOSTNAME', 'UNKNOWN') =~ /(\-(dev|staging|uat)\-)/i
+    include_person_details = Rails.env.development? || ENV.fetch('HOSTNAME', 'UNKNOWN') =~ /(-(dev|staging|uat)-)/i
```

Comment:
ooh, what's this, has regex syntax changed?

Reply:
the hyphen `-` was didn't need escaping after all